tests/test_other.py

```diff
@@ -9970,3 +9970,56 @@ def test_relocatable_limited_exports(self):
     self.assertGreater(len(exports_linkable), 1000)
     self.assertIn('sendmsg', exports_linkable)
     self.assertNotIn('sendmsg', exports)
+
+  @is_slow_test
+  def test_deps_info(self):
+    # Verify that for each symbol listed in deps_info all the reverse
+    # dependencies are indeed valid.
+    # To do this we compile a tiny test program that depend on the address
+    # of each function.  Once compiled the resulting JavaScript code should
+    # contain a reference to each of the dependencies.
+    start_at = None
+    assert not start_at or start_at in deps_info.deps_info
+    for function, deps in deps_info.deps_info.items():
+      if start_at:
+        if function == start_at:
+          start_at = None
+        else:
+          print(f'skipping {function}')
+          continue
+      create_test_file(function + '.c', '''
+      void %s();
+      int main() {
+        return (int)&%s;
+      }
+      ''' % (function, function))
+      cmd = [EMCC, function + '.c', '--profiling-funcs', '-Wno-incompatible-library-redeclaration']
```

Comment:
perhaps the command could set `REVERSE_DEPS=auto` to ensure we test the right thing even if we change that default?

Reply:
Done